addons/opsportal/1.0.x/opsportal-9.yaml

```diff
@@ -0,0 +1,52 @@
+---
+apiVersion: kubeaddons.mesosphere.io/v1beta1
+kind: Addon
+metadata:
+  name: opsportal
+  namespace: kubeaddons
+  labels:
+    kubeaddons.mesosphere.io/name: opsportal
+  annotations:
+    catalog.kubeaddons.mesosphere.io/addon-revision: "1.0.0-9"
+    appversion.kubeaddons.mesosphere.io/opsportal: "1.0.0"
+    endpoint.kubeaddons.mesosphere.io/opsportal: /ops/portal/
+    values.chart.helm.kubeaddons.mesosphere.io/opsportal: "https://raw.githubusercontent.com/mesosphere/charts/4155f480571eaf82c64ddd63d3d334b1105d0591/stable/opsportal/values.yaml"
+spec:
+  kubernetes:
+    minSupportedVersion: v1.15.6
+  cloudProvider:
+    - name: aws
+      enabled: true
+    - name: azure
+      enabled: true
+    - name: gcp
+      enabled: true
+    - name: docker
+      enabled: true
+    - name: none
+      enabled: true
+  chartReference:
+    chart: opsportal
+    repo: https://mesosphere.github.io/charts/stable
+    version: 0.2.5
+    values: |
+      ---
+      kommander-ui:
+        image:
+          tag: 2.26.1
```

Comment:
I am temporarily overriding this here because i forgot to update the chart. https://github.com/mesosphere/charts/blob/master/stable/opsportal/values.yaml#L25

I have created a jira ticket to update the chart and remove this override before 1.0. https://jira.d2iq.com/browse/D2IQ-64343

Reply:
:+1: